app/package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "react": "^16.9.0",
+    "bootstrap": "^4.3.1",
```

Comment:
Do we need `bootstrap`? It looks like you're only using `react-bootstrap`

Reply:
Yes, `react-bootstrap` requires a `bootstrap` stylesheet.  That is why they are installed together in the documentation (https://react-bootstrap.github.io/getting-started/introduction).

To check this, I uninstalled `bootstrap`, and the stylesheet I added to `app/src/index.js` was no longer recognized:

`Module not found: Can't resolve 'bootstrap/dist/css/bootstrap.min.css'`